xml_converter/generators/cpp_templates/compoundvalue.cpp

```diff
@@ -27,18 +27,19 @@ using namespace std;
     }
     return {{attribute_name}};
 }
-{% if attribute_variables[0].xml_export == "Parent" %}
-    string stringify_{{attribute_name}}({{class_name}} attribute_value) {
-        string output;
-        {% for n, attribute_variable in enumerate(attribute_variables) %}
-            {% if n == 0: %}
-                output = to_string(attribute_value.{{attribute_variables[n].attribute_name}});
-            {% else %}
-                output = output + "," + to_string(attribute_value.{{attribute_variables[n].attribute_name}});
-            {% endif %}
-        {% endfor %}
-        return output;
-    }
+{% if attribute_variables[0].xml_parent_export != [] %}
+
+string stringify_{{attribute_name}}({{class_name}} attribute_value) {
+    string output;
+    {% for n, attribute_variable in enumerate(attribute_variables) %}
+        {% if n == 0: %}
+            output = to_string(attribute_value.{{attribute_variables[n].attribute_name}});
+        {% else %}
+            output = output + "," + to_string(attribute_value.{{attribute_variables[n].attribute_name}});
+        {% endif %}
+    {% endfor %}
+    return output;
+}
```

Comment:
This should only export the fields that are in the xml_parent_export, right now it checks everything.

Reply:
This still needs to be done

Reply:
Done